next/src/components/Event.tsx

```diff
@@ -4,29 +4,57 @@ import Card from '@mui/material/Card';
 import CardActionArea from '@mui/material/CardActionArea';
 import CardContent from '@mui/material/CardContent';
 import Box from '@mui/material/Box';
-import TranscribeButton from 'components/TranscribeButton';
 import { EventProps } from 'types/EventProps';
+import { useState } from 'react';
+import Transcript from 'components/Transcript'
+import TranscribeButton from 'components/TranscribeButton';
+import ErrorMessage from './ErrorMessage';
 
 type Props = { event: EventProps }
 
 export default function Event(props: Props) {
+  const [transcript, setTranscript] = useState(props.event.transcript)
+  const [buttonDisplay, setButtonDisplay] = useState(transcript === "")
+  const [errorDisplay, setErrorDisplay] = useState(false)
+
   return (
     <Grid item xs={12} md={6}>
       <CardActionArea component='a' href='#'>
         <Card sx={{ display: 'flex' }}>
           <CardContent sx={{ flex: 1 }}>
             <Typography variant='subtitle1' color='text.secondary'>
-              { props.event.createdAt }
+              { formatDateTime(props.event.createdAt) }
             </Typography>
             <Typography component='h2' variant='h5'>
-              { props.event.title }
+              {props.event.title}
             </Typography>
             <Box mt={1.5}>
-              <TranscribeButton />
+              <Transcript transcript={transcript} />
+            </Box>
+            <Box mt={1.5}>
+              <TranscribeButton
```

Comment:
```suggestion
              { isTranscrip && <TranscribeButton
```
みたいな形で表示・非表示をコントロールできるかもです🙌

Reply:
なるほどです〜。他のところ(ErrorMessage や Transcript など)も、この書き方で良さそうですね。
変更してみます〜。